src/loaders/compute_tools/gtdb_tk/gtdb_tk.py

```diff
@@ -73,14 +72,30 @@ def _run_gtdb_tk(ids_to_files: Dict[Path, str], output_dir: Path, threads: int,
 
         for genome_id, classify_res in zip(summary_df[loader_common_names.GTDB_GENOME_ID_COL],
                                            summary_df[loader_common_names.GTDB_CLASSIFICATION_COL]):
+            genome_ids.add(genome_id)
             if classify_res.startswith(loader_common_names.GTDB_UNCLASSIFIED):
-                kbase_id = meta_dict[genome_id]
-                source_file_path = ids_to_files[genome_id]
                 error_message = f"GTDB_tk classification failed: {classify_res}"
-                fatal_tuples.append(FatalTuple(kbase_id, error_message, str(source_file_path), None))
+                fatal_tuple = create_gtdbtk_fatal_tuple(genome_id, ids_to_files, error_message)
+                fatal_tuples.append(fatal_tuple)
+    
+    miss_genome_ids = set(ids_to_files.keys()) - genome_ids
+    filtered_or_failed_genome_ids = get_filtered_or_failed_genome_ids(output_dir)
+    error_genome_ids = miss_genome_ids - filtered_or_failed_genome_ids
+
+    if error_genome_ids:
+        raise ValueError(
+            f"Miss {error_genome_ids} that are not found in either "
+            f"{loader_common_names.GTDB_FILTER_FILE_PATTERN} or "
+            f"{loader_common_names.GTDB_FAIL_GENOME_FILE}"
+        )
     
+    for miss_genome_id in miss_genome_ids:
+        error_message = f"No result for the genome {miss_genome_id}"
```

Comment:
Why not include the error message from the filtered or failed output files from GTDB_tk?

Reply:
Updated!

Reply:
:+1: